CHANGELOG.rst

```diff
@@ -41,7 +41,12 @@ Completions
   - ``otool``
   - ``mix phx``
   - ``neovim``
-  
+  - ``trash``
+  - ``trash-empty``
+  - ``trash-list``
+  - ``trash-put``
+  - ``trash-restore``
```

Comment:
Can you collapse these to one line ("`trash` and helper utilities `trash-empty`, `trash-put`, ...")?